src/askademic/main.py

```diff
@@ -8,6 +9,18 @@
 from rich.console import Console
 from rich.prompt import Prompt
 
+console = Console()
+
+# check this here because if not set the following imports will fail
```

Comment:
Can't think of another way than before these imports- if key isn't set it must fail before imports that need it